.ci/run-e2e-tests.sh

```diff
@@ -9,10 +9,17 @@ sed -i 's/imagePullPolicy: Always/imagePullPolicy: Never/g' test/operator.yaml
 export VERBOSE='-v -timeout 20m'
 if [ "${TEST_GROUP}" = "es" ]; then
     echo "Running elasticsearch tests"
+    export USE_OTEL_COLLECTOR=false
```

Comment:
Rather than explicitly setting it to `false` it would be better to enable it only when it's set to `true`.